otx/api/entities/metadata.py

```diff
@@ -59,6 +60,27 @@ def __eq__(self, other):
         return self.name == other.name and self.value == other.value and self.float_type == other.float_type
 
 
+class VideoMetadata(IMetadata):
+    """This class represents metadata of video.
+
+    Args:
+        video_id (ID): ID for video.
+        frame_idx (int): Index for frame.
+    """
+
+    def __init__(self, video_id: ID, frame_idx: int):
+        self.video_id = video_id
```

Comment:
Please set up the `name` field, like it's done for `FloatMetadata`.

Reply:
Do we need to implement this metadata as same with `FloatMetadata`?
I think `name` field is not needed for the `VideoMetadata`

Reply:
Ah, I'll add the `name` field according to the `IMetadata` (parent class)